Search.jsp

```diff
@@ -167,8 +172,37 @@ select * from article where title like '%<%=request.getAttribute("search2") %>%'
 </table>
 </div>
 
+<br><br>
+
+<sql:query var="result2" dataSource="${ db}">
+select * from comments where comment like '%<%=request.getAttribute("search2") %>%' order by time desc
+</sql:query>
+
+<div>
+<table class="table2">
+<tr>
+<th>Article Title</th>
+<th>Comment</th>
+<th>Time</th>
+<th>Author</th>
+</tr>
+<c:forEach items="${result2.rows}" var="row2">
+<c:set value="${ search2}" var="search2"></c:set>
+<tr>
+<td><a href="Controller?page=view-article&title=${ row2.title}"> <c:out value="${ row2.title}"></c:out> </a></td>
+<td><c:out value="${ row2.comment}"></c:out></td>
+<td><c:out value="${ row2.time}"></c:out></td>
+<td><c:set var = "string" value = "${ fn:length(row2.user)}"/><c:out value="${ fn:substring(row2.user, 0, 3)}***${ fn:substring(row2.user, string-8, string)}"></c:out></td>
+</tr>
+</c:forEach>
+<tr>
+</tr>
+</table>
+</div>

```

Comment:
What is purpose of this block?  Add a comment for that.

**Hui**

Reply:
I'll commit some comments soon.